scripts/src/shika.js

```diff
@@ -0,0 +1,21 @@
+function shika_send() {
+    const url = document.currentScript.getAttribute("url");
+    if (url === null) {
+        console.error("URL is not set");
+        return;
+    }
+    const id = document.currentScript.getAttribute("id") ?? "0";
```

Comment:
Conflicts with the `id` attribute (same as above) Also can we rename this to key here too?

Reply:
ah yeah forgot to do it here